packages/wrangler/src/d1/execute.tsx

```diff
@@ -48,6 +48,13 @@ export function Options(yargs: CommonYargsArgv) {
 			describe:
 				"Execute commands/files against a local DB for use with wrangler dev",
 			type: "boolean",
+			deprecated: true,
```

Comment:
Why deprecate this rather than removing it? Or at least hard failing if this appears?

Given that we are saying this is a beta command that can have breaking changes at any point, it seems unnecessary to have a deprecation period.

Better to just hard fail now with a decent error message and get people to update their stuff.

Reply:
Conversely if we are deprecating this with a warning, then how about a warning for people who are not providing any `--local` nor `--remote` and tell them "did you know that the behaviour has changed - you are now actually running locally here"?